incubator/elasticsearch/values.yaml

```diff
@@ -0,0 +1,18 @@
+# Default values for etcd.
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+Name: es
+PeerPort: 2380
+ClientPort: 2379
+Component: "elasticsearch"
+Image: "jetstack/elasticsearch-pet"
+ImageTag: "2.3.4"
+ImagePullPolicy: "Always"
+ClientReplicas: 2
+MasterReplicas: 2
+DataReplicas: 3
+DataCpu: "250m"
+DataMemory: "2Gi"
```

Comment:
Memory and CPU is usually different for client vs master vs data node.  How do we want to handle that?